### PR TITLE
Lets get build.sh once again building the images.  Setup mvn properties for each image name so that they can be easily changed.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

**Comment by rhuss** [comment]
cool, thx

File: pom.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
     <maven.exec.plugin.version>1.2.1</maven.exec.plugin.version>
     <clean-maven-plugin-version>2.4.1</clean-maven-plugin-version>
     <frontend-maven-plugin-version>1.6</frontend-maven-plugin-version>
-    <node.version>v6.11.5</node.version>
+    <node.version>v8.9.1</node.version>
```
Comment (gashcrumb) [inline pom.xml]: shh, don't tell anyone I snuck this in already :-)

Comment (chirino) [reply pom.xml]: don't think you can.. it breaks circleci .  see this build I had to cancel:
https://circleci.com/gh/syndesisio/syndesis/343

Comment (gashcrumb) [reply pom.xml]: Strange, this one worked -> https://circleci.com/gh/syndesisio/syndesis/340?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by rhuss** [comment]
Superseded by #493

**Pull request closed without merging**